service-config/src/test/java/com/palantir/config/service/example/AuthConfiguration.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.config.service.example;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.common.base.Optional;
+import com.palantir.config.service.ServiceConfiguration;
+import io.dropwizard.util.Duration;
+import org.eclipse.jetty.util.security.Password;
+import org.hibernate.validator.constraints.NotEmpty;
+import org.immutables.value.Value;
+import org.immutables.value.Value.Immutable;
+import org.immutables.value.Value.Style;
+
+/**
+ * Configuration for the Authentication.
+ */
+@Value.Immutable
```

Comment:
The other classes in this commit use `@Immutable` directly (omitting the "`Value`"). Should stick with a single style for consistency.